[live555] build failure

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
vcpkg install live555:x64-windows
```

### Failure logs

**Installing 1/1 live555:x64-windows@2024-10-31...
Building live555:x64-windows@2024-10-31...**
-- Note: live555 only supports static library linkage. Building static library.
Downloading live.2024.10.31.tar.gz
Failed to download live.2024.10.31.tar.gz.
If you are using a proxy, please ensure your proxy settings are correct.
Possible causes are:
1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable to `https//address:port`.
This is not correct, because `https://` prefix claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr, etc...) is an HTTP proxy.
Try setting `http://address:port` to both HTTP_PROXY and HTTPS_PROXY instead.
2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings set by your proxy software. See, https://github.com/microsoft/vcpkg-tool/pull/77
The value set by your proxy might be wrong, or have same `https://` prefix issue.
3. Your proxy's remote server is our of service.
If you've tried directly download the link, and believe this is not a temporay download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
to report this upstream download server failure.
error: http://live555.com/liveMedia/public/live.2024.10.31.tar.gz: failed: status code 404
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:231 (message):
  Download failed, halting portfile.
Call Stack (most recent call first):
  ports/live555/portfile.cmake:4 (vcpkg_download_distfile)
  scripts/ports.cmake:196 (include)

### Additional context

Need to update to the latest build

--- a/ports/live555/portfile.cmake
+++ b/ports/live555/portfile.cmake
@@ -4,7 +4,7 @@ string(REPLACE "-" "." format_version ${VERSION})
 vcpkg_download_distfile(ARCHIVE
     URLS "http://live555.com/liveMedia/public/live.${format_version}.tar.gz"
     FILENAME "live.${format_version}.tar.gz"
-    SHA512 88325E4B87F1ED8DE16AF08489A1B856E0526C627D5F590ED7060D8899A91EC9D9947A5FB0C4714A3ACB5F5F37A2EB5C63F59739A75DCBEA70A79263F7EB56AA
+    SHA512 ee2bf17d2803c4bb6f49408a123de9238273749b9c110113facbf78eb01b9961bbd04178335f40d36425c9f96a26ee3da57e970f86d4912b4ec42ab6f4b2c7e9
 )
 
 vcpkg_extract_source_archive(

--- a/ports/live555/vcpkg.json
+++ b/ports/live555/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "live555",
-  "version-date": "2024-10-31",
+  "version-date": "2024-11-28",
   "description": "A complete RTSP server application",
   "homepage": "http://www.live555.com/liveMedia",
   "license": "GPL-3.0-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5557,7 +5557,7 @@
       "port-version": 0
     },
     "live555": {
-      "baseline": "2024-10-31",
+      "baseline": "2024-11-28",
       "port-version": 0
     },
     "llfio": {

--- a/versions/l-/live555.json
+++ b/versions/l-/live555.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b83d9a10d5b979381dad2b345d934243ee5f507a",
+      "version-date": "2024-11-28",
+      "port-version": 0
+    },
+    {
       "git-tree": "d4a99bf379695cab7957b1ed8ec4a59c33b1dce0",
       "version-date": "2024-10-31",
       "port-version": 0

